### PR TITLE
add runAs CDI2 security tests

### PR DESCRIPTION
**Pull request by neilgsyoung**

### CODE REVIEW

**Comment by neilgsyoung** [comment]
fixes #1328

**Comment by neilgsyoung** [comment]
#build

**Comment by neilgsyoung** [comment]
#build

File: dev/com.ibm.ws.cdi.2.0_fat/fat/src/com/ibm/ws/cdi20/fat/tests/FATSuite.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2017, 2018 IBM Corporation and others.
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.2.0_fat/fat/src/com/ibm/ws/cdi20/fat/tests/FATSuite.java]: It should be 2018.

File: dev/com.ibm.ws.cdi.2.0_fat/test-applications/secureAsyncEventsApp/src/secureAsyncEventsApp/web/SecureAsyncEventsServlet.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2017, 2018 IBM Corporation and others.
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.2.0_fat/test-applications/secureAsyncEventsApp/src/secureAsyncEventsApp/web/SecureAsyncEventsServlet.java]: delete 2017

Comment (neilgsyoung) [reply dev/com.ibm.ws.cdi.2.0_fat/test-applications/secureAsyncEventsApp/src/secureAsyncEventsApp/web/SecureAsyncEventsServlet.java]: I think I need both dates, the classes were originally delivered under other PRs in 2017. I have now amended them in 2018.

**Pull request merged**